clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -495,8 +495,7 @@ protected void processMessageAndMaybeProduceToKafka(
       // only extension of IngestionTask which does a read from disk before applying the record. This makes the
       // following function
       // call in this context much less obtrusive, however, it implies that all views can only work for AA stores
-      int valueSchemaId =
-          rmdWithValueSchemaID != null ? rmdWithValueSchemaID.getValueSchemaId() : incomingValueSchemaId;
+      int valueSchemaId = mergeConflictResult.getValueSchemaId();
```

Comment:
Nitpick... can we just delete this variable and put `mergeConflictResult.getValueSchemaId()` inline, below. It's used in only one place from what I can tell.

Reply:
No wait, actually, this one need not change.

Reply:
It is the `incomingValueSchemaId` below that needs to be replaced...

Reply:
Ah you are right.  The incoming is what needs to be fixed.  But now I'm dubious about valueSchemaId given the lambda